cves/CVE-2018-6621.yml

```diff
@@ -238,8 +252,11 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer:  | 
+    The file edited was in the avcodec folder, and the file itself was the UT 
+    decoder file.
+  name: [avcodec, ut codec]
```

Comment:
I think avcodec is all that is needed here. Might want to double check this.

Reply:
It's specifically a member of the UT codec, which is a subsystem of the AV codecs.